backend/requirements/default.txt

```diff
@@ -12,7 +12,7 @@ distributed==2023.8.1
 fastapi==0.109.2
 fastapi-users==12.1.3
 fastapi-users-db-sqlalchemy==5.0.0
-filelock==3.12.0
+filelock==3.12.2
```

Comment:
I think we don't need to bump reqs right?

Reply:
i'll try downgrading and retesting, maybe an issue with my local setup, but was not able to get it working originally with regular req's

Reply:
this one is an issue with using Node 20 instead of Node 18, so packages will need to be upgraded at that point.   Can be downgraded in the meantime.